openerp/addons/base/tests/test_base.py

```diff
@@ -441,26 +441,6 @@ def test_101_create_translated_record(self):
         fr_context_cat = self.res_category.browse(cr, uid, self.cat_id, context={'lang':'fr_FR'})
         self.assertEqual(fr_context_cat.name, 'Clients', "Translation not found")
 
-    def test_102_duplicate_record(self):
```

Comment:
I still don't understand why this test should be removed. You should be able to duplicate a category; maybe the name has to be modified when copying.